Modelica/Electrical/Machines/Utilities/ParameterRecords/InductionMachineData.mo

```diff
@@ -30,7 +30,7 @@ record InductionMachineData "Common parameters for induction machines"
         2*pi*fsNominal/p) "Friction loss parameter record"
     annotation (Dialog(tab="Losses"));
   parameter Machines.Losses.CoreParameters statorCoreParameters(
-    final m=m,
+    m=m,
```

Comment:
I am not very familiar with these electrical machine models, but what I can see from my myopic perspective looks a bit surprising.  Could you please just explain why it would make sense to have a different `m` in the `statorCoreParameters` member than in the rest of the `InductionMachineData` record?

Reply:
Well we want to keep the parameters for core losses together, and the CoreParameter record is used for 
- DC machines (m=1)
- 3phase machines (Modelica.Electrical.Machines m=3)
- PolyPhase machines (Modelica.Magnetic.FundamentalWave, m>=3)
So we are instantiating the CoreParameter record in different machine parameter records, propagating the desired number of phases m. How should the CoreParameter record know which number of phases is used in the machine parameter record (one layer higher in hierarchy)? Or do I miss something?

Reply:
OK, I'll rephrase the question:  Why was `final` removed here?  Does any of your examples involve a `InductionMachineData` where the `m` differs from the `m` inside its own `statorCoreParameters`?

Reply:
Ok I understand - and you're right:
As long as I propagate final m=m from within the enclosing parameter record there's no need to give it free to change (and it is even better to restrict it). Reverted that.